### PR TITLE
Replaced logo with wordmark SVG, added artist credit

### LINKED ISSUE #28: [Bug] Add logo
**Describe the bug**

Valkey has now has a logo! 

valkey-logo-only

**Expected behavior**

The site should have a logo in the upper left hand corner in the header and should have favico set to the new logo. Ideally this would match the blue text color.

We should also [credit the logo creator](https://github.com/dizys) in the footer.

**Screenshots**

n/a

**Additional context**

n/a

### PR DESCRIPTION
**Pull request by kris-ms**
### Description

Replaces plain text logotype with SVG provided in assets folder. (/assets/img/Valkey-wordmark.svg)
Credits the creator of the new logo design.
 
### Issues Resolved

Closes #28 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

Cheers, please let me know if there are any issues!

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -8,7 +8,9 @@
 <body id="{{ page.sectionid }}" class="{{ page.body_class }}">
 <div class="header">
     <div class="width-limiter">
-        <a class="logo" href="/">Valkey</a>
+        <a href="/" class="logo">
+            <img src="/assets/img/Valkey-wordmark.svg" alt="Valkey Logo" width="180px" />
```
Comment (stockholmux) [inline _layouts/default.html:12]: For now this should probably just have the logo + the opensans "Valkey"

The wordmark was created before a bunch of stuff was settled: it's Valkey not ValKey and the font's hasn't been officially picked yet. 

I'd actually be okay if removed the wordmark file was removed for now.

**Review by stockholmux — CHANGES_REQUESTED** [review]
This is awesome! Thanks for the contribution. I have change in the review and after that it LGTM.

**Comment by kris-ms** [comment]
> This is awesome! Thanks for the contribution. I have change in the review and after that it LGTM.

Appreciate the review, changes attached, cheers!

**Review by stockholmux — APPROVED** [review]

**Comment by stockholmux** [comment]
@kris-ms Your work is live on Valkey.io 🎉

**Pull request merged**